.github/workflows/linux_conditional_compilation.yml

```diff
@@ -39,27 +39,28 @@ jobs:
       image: openvinogithubactions.azurecr.io/dockerhub/ubuntu:22.04
       volumes:
         - /mount/caches:/mount/caches
+      options: -e SCCACHE_AZURE_BLOB_CONTAINER -e SCCACHE_AZURE_CONNECTION_STRING
     env:
       DEBIAN_FRONTEND: noninteractive # to prevent apt-get from waiting user input
       CMAKE_BUILD_TYPE: 'Release'
       CMAKE_GENERATOR: 'Ninja Multi-Config'
-      CMAKE_CXX_COMPILER_LAUNCHER: ccache
-      CMAKE_C_COMPILER_LAUNCHER: ccache
+      CMAKE_CXX_COMPILER_LAUNCHER: sccache
+      CMAKE_C_COMPILER_LAUNCHER: sccache
       GITHUB_WORKSPACE: '/__w/openvino/openvino'
       OPENVINO_REPO: /__w/openvino/openvino/openvino
       INSTALL_DIR: /__w/openvino/openvino/openvino_install
       BUILD_DIR: /__w/openvino/openvino/openvino_build
       SELECTIVE_BUILD_STAT_DIR: /__w/openvino/openvino/selective_build_stat
       MODELS_PATH: /__w/openvino/openvino/testdata
-      CCACHE_DIR: /mount/caches/ccache/ubuntu22_x86_64_itt_clang_Release
+      SCCACHE_AZURE_KEY_PREFIX: ubuntu22_x86_64_itt_clang_Release
       CCACHE_TEMPDIR: /__w/openvino/openvino/ccache_temp
       CCACHE_MAXSIZE: 20G
```

Comment:
BTW, do we need CCACHE_TEMPDIR and CCACHE_MAXSIZE?
How cache size is controlled now?

Reply:
According to `sccache` [documentation](https://github.com/mozilla/sccache/blob/46f47cd88c44976ae5d8944dd642623b52c478f3/docs/Local.md?plain=1#L5), there are options only to control local folder cache. I'll remove `CCACHE_TEMPDIR` and `CCACHE_MAXSIZE` variables from the workflows.

The cache cleanup policy can be configured in Azure then https://learn.microsoft.com/en-us/azure/storage/blobs/lifecycle-management-overview